webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -158,24 +143,24 @@ const WorkspaceDetails: React.FC = () => {
   );
 
   const tabItems: TabsProps['items'] = useMemo(() => {
-    if (!workspace) {
-      return [];
-    }
-
-    const items: TabsProps['items'] = [
-      {
-        children: <WorkspaceProjects id={id} pageRef={pageRef} workspace={workspace} />,
-        key: WorkspaceDetailsTab.Projects,
-        label: 'Projects',
-      },
-      {
-        children: <TaskList workspace={workspace} />,
-        key: WorkspaceDetailsTab.Tasks,
-        label: 'Tasks',
-      },
-    ];
+    const items: TabsProps['items'] = Loadable.match(workspace, {
```

Comment:
**super nit:** there is a `Loadable.quickMatch` that allows you to define both the Loaded function and the NotLoaded result as function params to shorten these lines.
```
const items: TabsProps['items'] = Loadable.quickMatch(workspace, [], (w: Workspace) => {
  ...
});
```